Repairs/app/src/main/java/com/rooio/repairs/NavigationType.kt

```diff
@@ -0,0 +1,21 @@
+package com.rooio.repairs
+
+enum class NavigationType (private val value: String) {
+    DASHBOARD("DASHBOARD"),
+    SETTINGS("SETTINGS"),
+    EQUIPMENT("EQUIPMENT"),
+    JOBS("JOBS");
+
+    override fun toString(): String {
+        return value
+    }
+
+    fun getIntRepr(): Int {
```

Comment:
do we need int/string representation for the navigation types? I think it's fine by itself

Reply:
Good looks! I will change that!